canonical-web-and-design/charmhub.io
# 1861 add endpoint for embeddable cards

## Done
- _Add embeddable card endpoint and template._
- _Build embeddable cards page._
- _Fix a layout bug on the listing page._

## How to QA
- Run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/[charm_name]/publicise/cards
- Change the type of card by checking/un-checking things on the page. See it all works fine
- Go to http://localhost:8045/[charm_name]/listing and see the layout looks fine

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1861
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1862

## Screenshots
![image](https://user-images.githubusercontent.com/40214246/109674467-d8de5300-7b6e-11eb-8967-790e5c36c33b.png)
